app/controllers/idv/by_mail/enter_code_controller.rb

```diff
@@ -148,11 +148,8 @@ def user_did_not_receive_letter?
 
       def user_can_request_another_letter?
         return @user_can_request_another_letter if defined?(@user_can_request_another_letter)
-        gpo_mail = Idv::GpoMail.new(current_user)
-        @user_can_request_another_letter =
-          FeatureManagement.gpo_verification_enabled? &&
-          !gpo_mail.rate_limited? &&
-          !gpo_mail.profile_too_old?
+        policy = Idv::GpoVerifyByMailPolicy.new(current_user)
+        @user_can_request_another_letter = policy.resend_letter_available?
```

Comment:
153-155 became the `resend_letter_available?` method.

I am not convinced this memoization ever saves us anything, but 🤷